src/main/java/io/Output.java

```diff
@@ -0,0 +1,87 @@
+package io;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Output {
+
+    static OutputHandlerImpl handler;
```

Comment:
this should be static Output instance
you want to store an instance of Output class with an instance of OutputHandler stored inside